ArduPlane/ArduPlane.cpp

```diff
@@ -594,11 +594,11 @@ void Plane::update_alt()
             tecs_target_alt_cm = MAX(tecs_target_alt_cm, prev_WP_loc.alt - home.alt) + (g2.rtl_climb_min+10)*100;
         }
 
+        TECS_controller.set_tkoff_settings(get_takeoff_pitch_min_cd(), mode_takeoff.level_alt);
```

Comment:
i think it would be better to pass a min throttle, with an API that sets the min throttle in TECS for the next TECS update loop, auto-cleared. That would allow the decision to be made by the core plane code, with common code for mode takeoff and auto takeoff

Reply:
As requested, I have implemented `TECS::set_min_throttle()` and TAKEOFF mode uses it to command a max throttle climb during FlightStage::Takeoff.